pkgs/applications/audio/plexamp/linux.nix

```
@@ -0,0 +1,32 @@
+{ lib, fetchurl, appimageTools, pkgs, pname, version, meta, ... }:
+
+let
+  inherit pname version meta;
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://plexamp.plex.tv/plexamp.plex.tv/desktop/Plexamp-${version}.AppImage";
+    name = "${pname}-${version}.AppImage";
+    sha512 = "7NQmH42yzItReQ5WPdcbNPr/xed74H4UyDNlX5PGmoJRBpV1RdeuW1KRweIWfYNhw0KeqULVTjr/aCggoWp4WA==";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name pname version src;
```

```suggestion
    inherit pname version src;
```
providing name and pname+version is unecessary